translations/fr.yaml

```diff
@@ -160,7 +160,7 @@ general:
   learnerGroupTitleFilterPlaceholder: "Filtre par titre de groupe d'apprenants"
   learnerGroupTitlePlaceholder: "Ajoutez une titre pour cette groupe d'apprenants"
   learningMaterial: "Matière d’Étude"
```

Comment:
This line also needs to update to `Matériel d'Étude`

Reply:
good catch. it's the crooked apostrophe that caused me to miss this one. fixed.